tasks/install_from_source.yml

```diff
@@ -14,31 +14,43 @@
     checksum: "{{ sox_checksum | default(omit) }}"
     mode: 0644
 
+# The unarchive must change the permissions of the destination folder when mode parameter is supplied.
+# But it behaves differently with tar.gz and zip archives.
+# When extracting tar.gz, it doesn't touch the destination folder's permissions.
+# While in case of extracting zip into something like /tmp/ with mode: 755, it overrides the permissions of /tmp to 755.
+# This step will create a direcoty specific to this role to extract files to
+- name: Create directory for extracted files
+  file:
+    path: "{{ sox_download_dir }}/sox-{{ sox_version }}"
```

Comment:
@timothylytle Please remove the version from this temporary directory

Reply:
This has been updated